examples/utils/bundle-size-tests/src/experimentalSharedTree.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SharedTree } from "@fluid-experimental/tree";
+
+export function apisToBundle() {
+	return SharedTree;
```

Comment:
Use the same getFactory method.

Reply:
Here I am trying to follow the same pattern that new SharedTree bundle uses.

SharedTreeFactory ultimately creates an instance of SharedTree class, so it should be equivalent. Let me check it in as-is and see how it goes.

Reply:
Running the bundle report locally outputs the same numbers. So keeping this as is.